app/controllers/concerns/session_path_concern.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module SessionPathConcern
+  def transition_checker_new_session_url(**params)
+    "#{Plek.find('frontend')}/sign-in?#{params.compact.to_query}"
+  end
+
+  def transition_checker_new_session_callback_url(**params)
```

Comment:
The transition checker shouldn't be calling this URL.  The users browser gets redirected to it from the account-manager, and it redirects the user back to the transition checker.